src/membrane-keypad-element.ts

```diff
@@ -182,4 +187,26 @@ export class MembraneKeypadElement extends LitElement {
       );
     }
   }
+
+  private keyStrokeDown(key: string) {
+    const keyName = `key-${key.toLowerCase()}`;
+    const text = keyName[keyName.length - 1];
+    const selectedKey = this.shadowRoot?.querySelector(`[data-key-name="${keyName}"]`);
+    this.down(text, selectedKey as SVGElement);
+  }
+
+  private keyStrokeUp(key: string) {
+    let selectedKey = this.shadowRoot?.querySelector('.pressed');
+    const keyName = `key-${key.toLowerCase()}`;
+
+    if (key === 'Shift') {
+      this.shiftPressed = false;
```

Comment:
seems like `shiftPressed` is only set to false here, and never read. Perhaps we can remove it altogether?